README.md

```diff
@@ -1,131 +1,159 @@
-# Skybian
+Builds require:
```

Comment:
README should start with what the repo is/contains.

Reply:
yes

Reply:
changed and updated readme